build.gradle

```diff
@@ -48,7 +48,6 @@ subprojects {
     apply plugin: 'checkstyle'
     apply plugin: 'findbugs'
     apply plugin: 'maven-publish'
-    apply plugin: 'signing'
```

Comment:
I think `signing` related statements have been removed by mistake. This need to be fixed.